manage.py

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env python
 import sys
+import os
 
 
 if __name__ == "__main__":
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "admin.base.settings")
```

Comment:
This will break API, we should figure out how to run migrations without using this in the common `manage.py` file. I would suggest that we look into making an invoke script in `tasks.py` ala `adminserver` command.